### ISSUE TITLE
clang error when Cython is absent

### ISSUE THREAD

**Issue by isaacl** [aio-libs/aiohttp#231]
It's possible my system is not setup correctly...  OS X mavericks, tools from xcode 6.1.1.

```
building 'aiohttp._multidict' extension

clang -Wno-unused-result -fno-common -dynamic -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I/usr/local/include -I/usr/local/opt/sqlite/include -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.9.sdk -I/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.9.sdk/System/Library/Frameworks/Tk.framework/Versions/8.5/Headers -I/Users/ilevy/code/account-db/venv/include -I/usr/local/Cellar/python3/3.4.2_1/Frameworks/Python.framework/Versions/3.4/include/python3.4m -c aiohttp/_multidict.c -o build/temp.macosx-10.9-x86_64-3.4/aiohttp/_multidict.o

clang: error: no such file or directory: 'aiohttp/_multidict.c'

clang: error: no input files

error: command 'clang' failed with exit status 1
```

**Comment by asvetlov** [comment]
To build from github you should have Cython installed.
I want to include generated `aiohttp/_multidict.c` only in _distribution_, not in git tree.

Also I'll make _multidict.c compilation optional, pure python version will be used if user has no C compiler (think Windows).

**Comment by asvetlov** [comment]
Fixed in d6259e1
